apis/Google.Devtools.AspNet/Google.Devtools.AspNet/CloudTrace/StartSpanOptions.cs

```diff
@@ -19,7 +19,7 @@ namespace Google.Devtools.AspNet
     /// </summary>
     public sealed class StartSpanOptions
     {
-        public readonly SpanKind SpanKind;
+        public SpanKind SpanKind { get; }
```

Comment:
These still need /// comments. You should see warnings for these when building, so make sure the warnings list is clear.